### PR TITLE
fix: classify client states without consensus states as expired

### PR DESCRIPTION
**Pull request by timlind**
## Description

closes: #850 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/light-clients/07-tendermint/types/client_state.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func (cs ClientState) Status(
 	// get latest consensus state from clientStore to check for expiry
 	consState, err := GetConsensusState(clientStore, cdc, cs.GetLatestHeight())
 	if err != nil {
-		return exported.Unknown
+		return exported.Expired
```
Comment (colin-axner) [inline modules/light-clients/07-tendermint/types/client_state.go:81]:
```suggestion
                // if the client state does not have an associated consensus state for its latest height
                // then it must be expired
		return exported.Expired
```

Comment (colin-axner) [reply modules/light-clients/07-tendermint/types/client_state.go:81]: might need to commit locally, I don't know how to format properly with gh UI

**Review by colin-axner — APPROVED** [review]

**Review by AdityaSripal — APPROVED** [review]

**Comment by colin-axner** [comment]
Hey @timlind, there appears to be a broken test, I think the expected value just needs to be changed. Can you add a changelog as well in the state machine breaking section?

**Comment by colin-axner** [comment]
The branch needs to be brought up to date with main and then should automerge. Thanks again @timlind!

**Comment by marshsif** [comment]
Hi @colin-axner could you or someone on the team give us a sense of when this might be included in the next 2.x release? We'd like to be able to deploy the patch on our end at least, and see how things look. Thanks

**Comment by crodriguezvega** [comment]
Hi @marshsif. It is very likely that this will be released next week in v2.1.0.

**Comment by crodriguezvega** [comment]
Hi @marshsif. If v2.1.0 imports SDK 0.45.1, would that be a problem for you?

**Comment by marshsif** [comment]
No, I think that would be ok. At this point, we're going to deploy a forked version of IBCGO with the patch, and then when we do the v.45 upgrade on our end, we can include 2.1.

**Comment by marshsif** [comment]
Thanks again for keeping us updated on this, really appreciate it @crodriguezvega!

**Pull request merged**